star_formation_history/loadObservationalData.py

```diff
@@ -63,7 +63,8 @@ def read_obs_data(path="observational_data"):
         ObservationalData(obs2_a, obs2_rho, obs2_rho_err, "Karim et al. (2011) [radio]")
     )
 
-    # SFR Observational data from Bouwens 2012
+    # SFR Observational data from Bouwens 2012 (not used included by Bouwens 2015)
```

Comment:
What does this comment mean?

Reply:
The Bouwens et al. (2012) observations are included and with additional data this results in the Bouwens et al. (2015) data points, so in this version I commented out the Bouwens et al. (2012) data points.